### PR TITLE
Removing potential NPE

### PR DESCRIPTION
**Pull request by ebudai**
Another for #2135

---
This change is [Reviewable](https://reviewable.io/reviews/rptools/maptool/2146)

### CODE REVIEW

File: src/main/java/net/rptools/maptool/client/AppActions.java [hunk 1]
```diff
@@ -2470,9 +2470,12 @@ protected void done() {
 
         MapTool.setCampaign(campaign.campaign, campaign.currentZoneId);
         ZoneRenderer current = MapTool.getFrame().getCurrentZoneRenderer();
-        if (campaign.currentView != null && current != null)
-          current.setZoneScale(campaign.currentView);
-        current.getZoneScale().reset();
```
Comment (ebudai) [inline src/main/java/net/rptools/maptool/client/AppActions.java:2475]: current can be null here

File: src/main/java/net/rptools/maptool/client/MapToolLineParser.java [hunk 2]
```diff
@@ -1410,12 +1410,15 @@ public String getTokenLibMacro(String macro, String location) throws ParserExcep
    *     the caller doesn't have access to the token.
    */
   public Token getTokenMacroLib(String location) throws ParserException {
+    if (location == null) {
+      return null;
+    }
     if (!location.matches("(?i)^lib:.*")) {
       throw new ParserException(I18N.getText("lineParser.notALibToken"));
```
Comment (ebudai) [inline src/main/java/net/rptools/maptool/client/MapToolLineParser.java:1417]: location can be null here

File: src/main/java/net/rptools/maptool/client/MapToolLineParser.java [hunk 3]
```diff
@@ -1410,12 +1410,15 @@ public String getTokenLibMacro(String macro, String location) throws ParserExcep
    *     the caller doesn't have access to the token.
    */
   public Token getTokenMacroLib(String location) throws ParserException {
+    if (location == null) {
+      return null;
+    }
     if (!location.matches("(?i)^lib:.*")) {
```
Comment (ebudai) [inline src/main/java/net/rptools/maptool/client/MapToolLineParser.java:1416]: location can be null here

**Comment by Phergus** [comment]
___
*[src/main/java/net/rptools/maptool/client/ui/drawpanel/DrawPanelPopupMenu.java, line 408 at r5](https://reviewable.io/reviews/rptools/maptool/2146#-MDuGxy8DXjlP5gxhJa_:-MDuWWW77Z3VEq5ak3fF:bmwhfla) ([raw file](https://github.com/rptools/maptool/blob/a843301009744f72a2bfe5b4e74abab8ee7c52ef/src/main/java/net/rptools/maptool/client/ui/drawpanel/DrawPanelPopupMenu.java#L408)):*
<details><summary><i>Previously, ebudai (Eric Budai) wrote…</i></summary><blockquote>

This was more messed up than I thought.  It should have been continue in the block, and the VblTool() call outside it.
</blockquote></details>

Had me confused.  :)

**Comment by ebudai** [comment]
Campaign loading is broken somehow.  I'm fixing it now.

**Comment by ebudai** [comment]
I did a bunch of testing, it seems fine now.  I'll keep testing and pushing any fixes if I find anything.  Most of the issues come from application of `static`

**Comment by Phergus** [comment]
Makes sense.  That's what Craig was referring to earlier.

**Comment by ebudai** [comment]
I can find no other issues.

**Comment by Phergus** [comment]
Now if I can just get Reviewable to respond...

**Pull request merged**